platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/transients/field/WorkflowUserField.java

```diff
@@ -61,22 +63,65 @@ private List<String> getUserTaskAgeStrings(R row) {
 			.collect(Collectors.toList());
 	}
 
-	private Map<AGUser, DayTime> getUserTaskCreationMap(Set<AGWorkflowTask> tasks) {
+	private Map<UserAndDelegationUser, DayTime> getUserTaskCreationMap(Set<AGWorkflowTask> tasks) {
 
 		AGWorkflowTask.CREATION_TRANSACTION.prefetchAll(tasks);
 
-		TreeMap<AGUser, DayTime> userTaskCreationMap = new TreeMap<>();
+		Map<UserAndDelegationUser, DayTime> userTaskCreationMap = new HashMap<>();
 		for (AGWorkflowTask task: tasks) {
 			Optional<AGTransaction> creationTransaction = Optional.ofNullable(AGWorkflowTask.CREATION_TRANSACTION.getValue(task));
-			if (creationTransaction.isPresent()) {
-				userTaskCreationMap.merge(task.getUser(), creationTransaction.get().getAt(), DayTime::min);
+
+			if (creationTransaction.isPresent() && !task.hasActiveDelegation()) {
+				userTaskCreationMap.merge(new UserAndDelegationUser(task.getUser(), task.getUser()), creationTransaction.get().getAt(), DayTime::min);
+			} else if (task.hasActiveDelegation()) {
+				AGWorkflowTaskDelegation delegation = task.getActiveDelegation().get();
+				Optional<DayTime> lastModificationTimeFormLog = delegation.getLastModificationTimeFormLog();
+				if (lastModificationTimeFormLog.isPresent()) {
+					userTaskCreationMap
+						.merge(new UserAndDelegationUser(task.getUser(), delegation.getTargetUser()), lastModificationTimeFormLog.get(), DayTime::min);
+				}
 			}
+
 		}
 		return userTaskCreationMap;
 	}
 
-	private String getUserTaskAgeString(AGUser user, int days) {
+	private String getUserTaskAgeString(UserAndDelegationUser userAndCurrentDelegation, int days) {
+
+		if (userAndCurrentDelegation.getUserDefinedInTask() == userAndCurrentDelegation.getUserInCharge()) {
```

Comment:
I think this check should be a method on the private class, e.g.
```
public boolean isDelegated() {
    return userDefinedInTask == userInCharge;
}
```

Reply:
I see no clear advantage here, but OK